RFC/src/Glossary.md

```diff
@@ -161,6 +170,11 @@ A Mining Server is responsible for constructing new blocks by bundling transacti
 
 A Mining Worker is responsible for performing Proof-of-Work tasks received from its parent [Mining Server].
 
+## Node id
```

Comment:
```suggestion
## Node ID
```

Reply:
I think we should capitalize ID in the text to be inline with its most common usage.

Reply:
Change applied in new PR.